pkg/artifactory/resource_artifactory_managed_user_test.go

```diff
@@ -0,0 +1,92 @@
+package artifactory
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccManagedUser(t *testing.T) {
+	const userFull = `
+		resource "artifactory_managed_user" "%s" {
+			name        		= "dummy_user%d"
+			email       		= "dummy%d@a.com"
+			password			= "Password1"
+			admin    			= true
```

Comment:
Looks like few of the resource's key value pairs are not aligned.

Reply:
Look good here (and in my editor). Can you point out the issue?

Reply:
For me ... the "=" for password and admin fields are not in alignment with the rest ... ie: name, email.

Reply:
![Screen Shot 2022-04-12 at 10 53 02 AM](https://user-images.githubusercontent.com/27263/163023958-83d00703-6486-4f83-81f7-ee73c6fcc865.png)
<img width="463" alt="Screen Shot 2022-04-12 at 10 53 09 AM" src="https://user-images.githubusercontent.com/27263/163023975-30f35068-0cb0-4275-a553-7c50ae635bea.png">

Look fine to me. Is your editor/IDE setup correctly with tab/spaces?

Reply:
I am noticing this when reviewing the PR in GH webpage not from my IDE/editor. 
You can ignore if needed ... it is just cosmetic anyways.